blueprint.js

```diff
@@ -198,12 +198,25 @@ $Element.prototype.copy = function() {
 			res.childNodes.push(child.clone());
 		}
 	}
+	res.data = Object.assign({}, this.data);
 	// Return the copy
 	return res;
 }
 
+$Element.prototype.prepare = function(callback) {
+	this.data.prepare = callback;
```

Comment:
Can you change this to push to an array of preparation callbacks?

Reply:
Ooooooo... having an array of callbacks sounds handy, especially if you want to have callbacks you reuse between blueprints.